integration_tests/src/main/python/sort_test.py

```diff
@@ -55,6 +55,70 @@ def test_single_orderby(data_gen, order):
     assert_gpu_and_cpu_are_equal_collect(
             lambda spark : unary_op_df(spark, data_gen).orderBy(order))
 
+<<<<<<< HEAD
```

Comment:
unresolved merge conflict accidentally checked in

Reply:
Thank you! I will remove these.